openedx/features/caliper_tracking/tests/tests.py

```diff
@@ -30,7 +29,13 @@ class CaliperTransformationTestCase(TestCase):
         return_value='honor',
         autospec=True
     )
-    def test_caliper_transformers(self, mock_function):
+    @mock.patch(
+        'openedx.features.caliper_tracking.utils.get_team_url_from_team_id',
+        return_value="http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/teams/" +
```

Comment:
Please use appropriate methods to create multiline strings.

https://stackoverflow.com/questions/10660435/pythonic-way-to-create-a-long-multi-line-string

Reply:
Resolved